test/x8-lut.cc

```diff
@@ -2,1583 +2,53 @@
 //
 // This source code is licensed under the BSD-style license found in the
 // LICENSE file in the root directory of this source tree.
-//
-// Auto-generated file. Do not edit!
-//   Specification: test/x8-lut.yaml
-//   Generator: tools/generate-lut-test.py
-
 
 #include <gtest/gtest.h>
 #include "xnnpack/common.h"
 #include "xnnpack/isa-checks.h"
 #include "xnnpack/lut.h"
 #include "lut-microkernel-tester.h"
```

Comment:
If this is the only use of this header, just move the contents of it (and any associated .cc files) to be in this file directly.